app/data/events.json

```diff
@@ -1,4 +1,11 @@
 [
+	{
+		"date": "2025-01-16T18:00:00.000-04:00",
+		"link": "https://www.eventbrite.com/e/boston-ts-club-new-year-new-type-tickets-1086018734269",
+		"location": "Microsoft NERD Center",
+		"time": "5:30pm - 8:30pm",
+		"topics": []
```

Comment:
I'm planning on giving a What's New in TypeScript talk.

Reply:
Yay!!